misc.c

```diff
@@ -3063,3 +3063,28 @@ lib_contains_symbol(const char *path, const char *s)
 	return ret;
 #endif /* HAVE_NLIST_H */
 }
+
+char *
+parse_ssh_string(const char *msg)
```

Comment:
Oh... You are right... Using notifier this is not needed anymore indeed. I think I had tested this but very likely with an old compiled-ssh build.

So all this can be dropped now.